app/react-native/src/bin/storybook-start.js

```diff
@@ -48,61 +39,3 @@ server.listen(...listenAddr, err => {
     process.exit(0);
   }
 });
-
-if (!program.skipPackager) {
-  let symlinks = [];
-
-  let roots = [projectDir];
-
-  if (program.root) {
-    roots = roots.concat(program.root.split(',').map(root => path.resolve(root)));
-  }
-
-  try {
-    // eslint-disable-next-line global-require
-    require('babel-register')({
-      presets: [require.resolve('babel-preset-flow')],
```

Comment:
do `babel-register` and `babel-preset-flow` still need to be installed?

Reply:
Good question. What were they used for in the first place?

Reply:
I have no idea =)

Reply:
Ah but it is flow and some other things. I will remove it. 
Thanks @igor-dv .